src/Components/Backoffice/CustomAppBar.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { AppBar, Box, Toolbar, IconButton, Avatar } from '@mui/material';
+import MenuIcon from '@mui/icons-material/Menu';
+
+const CustomAppBar = ({ user }) => {
```

Comment:
BackofficeAppBar maybe its a better name or if just a generic CustomAppBar it should be on Components/CustomComponents/ directory